lib/ui/window.dart

```diff
@@ -1350,7 +837,14 @@ enum Brightness {
 /// reasonable for a future of Flutter where we legitimately want to select an
 /// appropriate implementation at runtime.
 ///
-/// The only place that `WidgetsBinding.instance.window` is inappropriate is if
-/// a `Window` is required before invoking `runApp()`. In that case, it is
-/// acceptable (though unfortunate) to use this object statically.
-final Window window = Window._();
+/// The only place that using `WidgetsBinding.instance.platformDispatcher` is
+/// inappropriate is if access to these APIs is required before invoking
+/// `runApp()`. In that case, it is acceptable (though unfortunate) to use the
+/// [PlatformDispatcher.instance] object statically.
+///
+/// See also:
+///
+///  * [PlatformDispatcher.views], contains the current list of Flutter windows
+///    belonging to the application, including top level application windows
+///    like this one.
+final Window window = Window._(0, PlatformDispatcher.instance);
```

Comment:
Is the plan to deprecate this at some point?

Reply:
No, we really can't: too many things access it still. I would if I could. The plan is to change the type of this to `SingletonFlutterWindow`, but leave it here, and it will still be called `window`.

Then, once `Window` is no longer used, we can reuse it for the name of a widget that manages windows.